site/_apps/glimmer-hn-pwa.txt

```diff
@@ -0,0 +1,26 @@
+---
+weight: 16
+title: 'Glimmer HN'
+github-title: 'mhadaily/glimmer-hn-pwa'
+libraries:
+    - name: 'Glimmer.js'
+scaffolding: 'Ember CLI'
+module-bundling: 'Broccoli'
+service-worker: 'Application Shell + data caching with ESW'
+perfomance-patterns: 'Prefetch/Preload JS + DNS, Critical CSS, HTTP/2 Server Push'
+server-side-rendering: 'None'
+api: 'Node-hnapi (unofficial)'
+hosting: 'Firebase'
+authors:
+    - name: 'Majid'
+lighthouse: '91/100'
+interactive-em: '3.4s'
+interactive-faster-3g: '2.4s'
+lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170612_BV_5cd2c04584152647e2e556f5af8f5eba&run=3
+wpt-em-link: https://www.webpagetest.org/result/170708_DK_bcd8d8c3da33a5f7badc29417f3b95c7/
+wpt-faster-3g-link: https://www.webpagetest.org/result/170708_34_1ae3fbe0c7f0f90ad0b46282ab2ce5a5/
```

Comment:
New run through WPT for faster 3G:
https://www.webpagetest.org/result/170719_6Y_4ebdb9a788a0691f0c5159e40286545b/
interactive-faster-3g: 5.120s

Lighthouse-link: https://www.webpagetest.org/lighthouse.php?test=170719_6Y_4ebdb9a788a0691f0c5159e40286545b&run=2